src/Post_Command.php

```diff
@@ -143,6 +143,10 @@ public function __construct() {
 	 *     # Create post with content from given file
 	 *     $ wp post create ./post-content.txt --post_category=201,345 --post_title='Post from file'
 	 *     Success: Created post 1922.
+	 *
+	 *     # Create a post with multiple meta values.
+	 *     $ wp post create --post_title='A post' --post_content='Just a small post.' --meta_input='{"key1":"value1","key2":"value2"}
```

Comment:
The `'` is not balanced here.

`--meta_input='{"key1":"value1","key2":"value2"}'`

Reply:
This has been fixed a while ago in #147.